include/trick/VariableServerThread.hh

```diff
@@ -426,9 +449,19 @@ namespace Trick {
             int transmit_file(std::string file_name);
 
             /**
-             @brief Called by write_data to write data to socket in var_binary format.
+             @brief Called by write_data to write given variables to socket in var_binary format.
+            */
+            int write_binary_data( int Start, char *buf1, const std::vector<VariableReference *>& givenVars, VS_MESSAGE_TYPE messageType);
```

Comment:
I know Trick is inconsistent with variable names, but let's go ahead and call ```givenVars``` ```given_vars``` unless there's a good reason to use camel case here.

Reply:
Fixed this and some other camel case variables that I saw